scripts/run_e2e_local.sh

```diff
@@ -8,8 +8,7 @@ set -e
 timestamp=$(date +%s)
 namespace="e2e-tests-${timestamp}-$RANDOM"
 
-tmpdir=`mktemp -d 2>/dev/null || mktemp -d -t 'valuetmpdir'`
-cp test/e2e/e2e-values.yaml ${tmpdir}/e2e-values.yaml
```

Comment:
If you're looking at this line in the only one commit at a time view, I made a mistake here with removing this line. But it's rectified in the last commit.